_i18n/en.yml

```diff
@@ -127,12 +127,32 @@ hangouts:
     monero-site: Where the development of this website is coordinated.
 
 merchants:
-  intro1: Merchants of all kinds have come to value the financial privacy that Monero brings. Below is a list of the merchants that we know of that currently accept Monero for their goods and services. <b>These merchants are not vetted by the Getmonero team, see the disclaimer at the bottom of this page</b>. If a company no longer accepts Monero or you would like your business to be listed, please
-  intro2:  open a GitHub issue and let us know
-  intro3: (select the "Merchant" template and fill up all sections).
+  descr: On this page we list merchant directories, where you can spend your Moneroj and P2P, decentralized and centralized exchanges.
+  useit: "Use Monero!"
+  use: Over the years, merchants of all kinds have come to value the financial privacy that Monero brings. In fact, merchant adoption is continuously rising.
+  dirdescr: "If you are looking for places to spend your Monero, the directories below have some options. Please remember to do your own research before trusting them with your money:"
+  descrp2p: The best way to exchange Monero for other currencies (or the other way round) is using P2P exchanges, because they allow traders to directly interact with each other, without the need of trusted third parties (except maybe for providing an escrow service). These exchanges are more private then their centralized counterparts and allow people to be in complete control of their coins if they are non-custodial, but they usually have less volume then 'traditional' exchanges. In the section below, we offer a list of P2P exchanges that support Monero.
```

Comment:
```suggestion
  descrp2p: The best way to exchange Monero for other currencies (or the other way round) is using P2P exchanges, because they allow traders to directly interact with each other, without the need of trusted third parties (except maybe for providing an escrow service). These exchanges are more private then their centralized counterparts and allow people to be in complete control of their coins if they are non-custodial, but they usually have less volume than 'traditional' exchanges. In the section below, we offer a list of P2P exchanges that support Monero.
```